### ISSUE TITLE
integer-align the SVGs

### ISSUE THREAD

**Issue by Pomax** [flukeout/mozmaker-icon-font#2]
Right now the SVGs use fractional coordinates, which makes both the SVG base files and the font bigger than they need to be - it might be worth translating-scaling the coordinate system a bit so that (almost) all coordinates are integer

**Comment by flukeout** [comment]
Will look into this in sketch! I tried to build it pixel-aligned in Illustrator, but I think ultimately I changed the scaling in Sketch.

**Comment by Pomax** [comment]
yeah it's weird that sketch changes the actual coordinates instead of just using a master `<g transform="scale(...)">` there O_o

**Comment by flukeout** [comment]
Is this something that cool svg cleanup tool we've heard about could help with?

**Comment by Pomax** [comment]
You mean [svgo](https://github.com/svg/svgo)? It's pretty excellent but I don't know if it has a "find me the most integer-fitting coordinate solution"... that said it's open source so if it doesn't we can probably request it!
